testdroid_cmdline.sh

```diff
@@ -352,10 +353,11 @@ function get_result_files {
 #   test_run_id
 #   device_run_id
 # Returns:
-#   Void (writes files to subfolder 'results/')
+#   Void (writes files to subfolder TEST_RESULTS_DIR)
 #########################################
 function get_device_result_files {
-  mkdir -p "results"
+  rm -rf $TEST_RESULTS_DIR
```

Comment:
You can make `rm -rf` a bit more safe when used with variables: `rm -rf "${TEST_RESULTS_DIR:?}"`

quotes around variable: http://www.tldp.org/LDP/abs/html/quotingvar.html
`${VAR:?}` will check that variable is assigned and exit if it is not. http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_10_03.html